### DISCUSSION TITLE
Image sorting in the UI is unclear(looks like that is sorted by tag)

### DISCUSSION THREAD

**Discussion by tal-hason** [akuity/kargo#1019] [General]
In the UI in the images section, the images tag is sorted, it looks like in descending order from the tag value.

if not using semantic versioning in the image tag it can be kind of confusing.

![image](https://github.com/akuity/kargo/assets/97454240/41ade8dd-dd93-4a8a-97bf-3ef510591a36)
 
it should be sorted by, or freight time & date, or image time & date, so the order will be absolute from the new to the old.

**Comment by krancour** [comment]
> it should be sorted by, or freight time & date

That probably wouldn't work out in many cases. In your screen shot above, you subscribe to three image repos and a git repo. Suppose one image didn't have a new release for months and months. There many be many pieces of Freight, each with their own timestamp, referencing that one unchanged image.

> image time & date

That seems it would work better, but we don't currently capture that information. We should. I will look and see how easy (or not) it would be to obtain it.

**Reply by tal-hason** [reply]
Hi @krancour, have you had the opportunity to check this?

**Reply by krancour** [reply]
#1058 is a pre-req for it. Prior to that, we don't have date information on image tags. Or, more accurately, we never save it, but it isn't always available. After #1058, it will be available and we can save it if we plan to make use of it.

**Comment by krancour** [comment]
I think this was resolved long ago.
